runtime/doc/cmdline.txt

```diff
@@ -820,7 +820,7 @@ These modifiers can be given, in this order:
 		separator is removed.  Thus ":p:h" on a directory name results
 		on the directory name itself (without trailing slash).
 		When the file name is an absolute path (starts with "/" for
-		Unix; "x:\" for MS-DOS and WIN32), that part is not removed.
+		Unix; "x:\" for Windows), that part is not removed.
```

Comment:
:+1: I'm glad to see that we're referring to Windows platforms in a more unified way.